.github/ISSUE_TEMPLATE/pg-auto-failover-enterprise-release-checklist-template.md

```diff
@@ -0,0 +1,62 @@
+---
+name: pg-auto-failover-enterprise release checklist template
+about: Please make sure to follow the checklist below along the release process.
+title: Pg Auto Failover Enterprise release checklist - [vX.Y.Z]
+labels: release_checklist
+assignees: gurkanindibay
+
+---
+
+## Several things to note here
+
+These instructions assume you have `$VERSION`, `$PROJECT`, and `$REPO` environment variables set in your shell (e.g. `10.0.2`, `citus`, and `citus`). With those set, code from most steps can be copy-pasted.
+
+**After this checklist, you're still not done: open a release checklist for Enterprise too!**
```

Comment:
This is not valid here. Is it a copy/paste error?

Reply:
You're right Fixed